src/graphql/resolvers.ts

```diff
@@ -1,8 +1,14 @@
 import merge from "deepmerge";
 
+import { historyResolvers } from "./history/resolvers";
 import { launchesResolvers } from "./launches/resolvers";
 import { rocketsResolvers } from "./rockets/resolvers";
+import { ResolversTypes } from "@src/@types/graphql-schema";
 
-const resolvers = merge(launchesResolvers, rocketsResolvers);
+const resolvers = merge.all<ResolversTypes>([
+  launchesResolvers,
+  rocketsResolvers,
+  historyResolvers,
+]);
```

Comment:
I had to add .all() since basic merge function just takes two objects as arguments, and .all() can handle an array of several objects.